src/richText.js

```diff
@@ -90,13 +90,19 @@ define([
                 labelText.find('output').replaceWith(function () {
                     return extractXPathInfoFromOutputValue($(this).attr('value')).reference;
                 });
-                $(this.dragHandlerContainer.$).children("img").stickyover({
-                    title: displayId + '<small>' + xpath + '</small>',
+                // Remove ckeditor-supplied title attributes, which will otherwise override popover title
+                $(this.dragHandlerContainer.$).children("img").removeAttr("title");
+                // TODO: check with emord this can use popover...and if so, remove stickyover altogether
+                $(this.dragHandlerContainer.$).children("img").popover({
+                    trigger: 'hover',
+                    container: 'body',
+                    placement: 'bottom',
+                    title: '<h3>' + displayId + '</h3><div class="text-muted">' + xpath + '</div>',
```

Comment:
Should `xpath` be HTML-escaped here? I see it wasn't before, but looks like a possible opportunity for HTML injection.